robottelo/test.py

```diff
@@ -481,97 +509,97 @@ def _write_raw_csv_file(self, raw_file_name, time_result_dict,
 
             # for each thread, write its head and data
             for i in range(current_num_threads):
-                writer.writerow([
-                    'client-{}'.format(i)
-                ])
+                writer.writerow(['client-{}'.format(i)])
                 writer.writerow(time_result_dict.get('thread-{}'.format(i)))
             writer.writerow([])
 
-    def _write_raw_att_csv_file(self, raw_file_name, time_result_dict,
-                                current_num_threads, test_case_name):
-        """Write raw timing att results to csv file"""
-        self.logger.debug("Timing result is: {}".format(time_result_dict))
-        with open(raw_file_name, 'a') as handler:
-            writer = csv.writer(handler)
-            writer.writerow([test_case_name])
+        # generate line chart of raw data
+        test_category = self._get_output_filename(raw_file_name)
+        generate_line_chart_raw(
+            time_result_dict,
+            'Candlepin Subscription Raw - '
+            '({0}-{1}-clients)'
+            .format(test_category, current_num_threads),
+            '{0}-{1}-clients-raw-data-line-chart.svg'
+            .format(test_category, current_num_threads)
+        )
 
-            # for each thread, write its head and data
-            for i in range(current_num_threads):
-                writer.writerow(['client-{}'.format(i)])
-                for index in range(3):
-                    writer.writerow(
-                        time_result_dict.get('thread-{}'.format(i))[index])
-                    writer.writerow([])
-
-    def _write_stat_csv_file(self, stat_file_name, time_result_dict,
-                             current_num_threads, test_case_name, is_attach):
-        """Generate statistical result of concurrent ak/del/att"""
+    def _write_stat_csv_chart(self, stat_file_name, time_result_dict,
+                              current_num_threads, test_case_name):
```

Comment:
Nitpick: argument layout. I've given several other examples of how to rewrite code like this elsewhere in this PR.